views/js/qtiItem/core/interactions/Interaction.js

```diff
@@ -98,6 +98,11 @@ define([
             _.each(choices, function(choice){
                 if(Element.isA(choice, 'choice')){
                     try{
+                        if (renderer.name == 'xmlRenderer') {
+                            if ((new RegExp('<|>', 'g')).test(choice.text)) {
+                                choice.text = _.escape(choice.text);
+                            }
+                        }
```

Comment:
This is not good... 
views/js/qtiItem/core/ is the common qtiItem library and xmlRender is a specific implementation of its renderer.
If that fix need to be done, it should be on the views/js/qtiXmlRenderer